sonar.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo "Building docker image"
+docker build -f ./config/sonar.dockerfile -t sonar ./config/
+echo "Running docker container"
+docker run -dit -p 9000:9000 --name sonar sonar:latest
```

Comment:
please split this file into two to let Wercker and any other docker based CI launch the same sh files as user.
All commands above this line are required only for manual execution on local(by human).